radar.csv

```diff
@@ -108,7 +108,14 @@ Physical whiteboards,adopt,tools,true,"Some teams use physical whiteboards for i
 "Scrum",adopt,techniques,true,"Agile framework focused on time-boxed, iterative plannning. We default to this approach to encourage iterative delivery in teams and a simple process to follow."
-"Kanban",explore,techniques,true,"Flow-based agile framework the emphasises limiting work-in-progress (WIP). Historically adopted by teams who find time-boxed iterations restrictive, or need to work in a more reactive manner. Currently being explored by ICloneable<Sheep>."
+"Kanban",explore,techniques,true,"Flow-based agile framework the emphasises limiting work-in-progress (WIP). Historically adopted by teams who find time-boxed iterations restrictive, or need to work in a more reactive manner. Currently being explored by ICloneable&lt;Sheep&gt;."
 "MoSCoW",retire,techniques,true,"MoSCoW stands for Must, Should, Could and Wont. This prioritisation technique requires the majority of features / stories to be identified up front - this does not fit well with our exploratory approach to finding customer needs and drilling in based on our riskiest assumption. It also has the effect of creating fixed view of what will be delivered and can result in teams delivering work that is not valued by customers."
 "master is always releasable",adopt,techniques,true,"'master as always releasable' is an ethos teams will follow and constantly strive to make true. We should have enough confidence in the product and automated tests that anyone can press the release button at any time. We release whenever we have something to release. We should act like master is always releasable at any time, and when we discover that it isn't/wasn't releasable, reflect on why and add the appropriate checks for next time."
-"Zero bugs policy",explore,techniques,true,"A <a href='https://medium.com/qualityfaster/the-zero-bug-policy-b0bd987be684'>zero-bug policy</a> does not mean bug-free code production; it means striving to eradicate all known bugs. ICloneable<Sheep> will be exploring this with support from the Quality Coaches."
+"Zero bugs policy",explore,techniques,true,"A <a href='https://medium.com/qualityfaster/the-zero-bug-policy-b0bd987be684'>zero-bug policy</a> does not mean bug-free code production; it means striving to eradicate all known bugs. ICloneable&lt;Sheep&gt; will be exploring this with support from the Quality Coaches."
+"Karma",adopt,languages and frameworks,true,"<a href='https://karma-runner.github.io/latest/index.html'>Karma</a> is a test runner for JavaScript which spawns a web server that executes source code against test code for each of the browsers connected."
+"jasmine",explore,languages and frameworks,true,"<a href='https://jasmine.github.io/'>Jasmine</a> is a BDD framework for testing JavaScript code. The Figleaf team is evaluating this."
```

Comment:
Sorry, realized the way I wrote this in the issue was a bit misleading - Figleaf isn't currently using jasmine, but Monitor are using it through Karma.